index.html

```diff
@@ -8,8 +8,9 @@
 
     <link rel="stylesheet" type="text/css" href="normalize.css" />
     <link rel="stylesheet" type="text/css" href="styles.css" />
-
-    <script defer src='functions.js'></script>
+    
+    <script defer src="jQuery.js"></script>
```

Comment:
do you really want jQuery for 1 fadeIn? It is slowing your page load down by 25%

Reply:
We are happy with the current load time as this only adds 0.1 seconds which is not unreasonable. This could be changed in a future sprint if user/product owner notices this is a hinderance.